### ISSUE TITLE
Suddenly blank map is displayed for all KnownTileSources

### ISSUE THREAD

**Issue by HossZamani** [Mapsui/Mapsui#2410]
**Mapsui Version**
Mapsui 4.1.2

**Mapsui Platform**
WPF
Haven't tested any other platform

**Device**
Windows - Windows 10 Pro 22H2 (OS Build 19045.3803)

**Describe the bug**
I had OpenStreetMaps displaying in my WPF window. Not sure how this can trigger anything but I copied that window to make another window and since then the map on the original window is showing blank. I've deleted the new window from my project, cleaned and rebuilt and still getting the same problem. I've also checked #2172 but it doesn't seem relevant as I have used other KnownTileSources and still getting the same issue. Not sure if this helps but I can see the "OpenStreetMaps contributors" (or "Microsoft" when using Bing) at the bottom of the control but no map is displayed. 

**To Reproduce**
Not sure how to reproduce this as it has suddenly stopped working without any specific code change in my project. 

**Expected behavior**
Map to be displayed as usual.

**Screenshots**
![image](https://github.com/Mapsui/Mapsui/assets/27878288/c0f1db58-5d0d-46d8-be00-4731e5aa1860)
![image](https://github.com/Mapsui/Mapsui/assets/27878288/2f8c1408-e915-4d83-8ef7-f165c5c2f175)
![image](https://github.com/Mapsui/Mapsui/assets/27878288/4b885cf6-d510-4c8e-a5fb-f31d90ee8198)

**Additional context**
I tried using the Logger to find additional info and I'm getting the following Error:
"Unexpected error in skia renderer"; 
Message: "Method not found: 'SkiaSharp.SKPicture SkiaSharp.SKPicture.Deserialize(System.ReadOnlySpan`1<Byte>)'."
StackTrace: "at Mapsui.Rendering.Skia.BitmapHelper.LoadBitmap(Object bitmapStream)\r\n   at 
Mapsui.Rendering.Skia.RasterStyleRenderer.Draw(SKCanvas canvas, Viewport viewport, ILayer layer, IFeature feature, IStyle style, IRenderCache renderCache, Int64 currentIteration)"

**Comment by pauldendulk** [comment]
Thanks for the bug report. The logged error is helpful. This does not look like a bug in Mapsui. There appears to be something wrong with the nuget references, check the versions you are using, or perhaps something corrupted in the build. Delete bin and obj folders, and perhaps the nuget cache on your machine.

**Comment by charlenni** [comment]
Regarding the empty maps: Are there any problems with the network? My anti-virus software sometimes says, that this new app has no network access until you allow it. Then all maps working.

**Comment by HossZamani** [comment]
> Thanks for the bug report. The logged error is helpful. This does not look like a bug in Mapsui. There appears to be something wrong with the nuget references, check the versions you are using, or perhaps something corrupted in the build. Delete bin and obj folders, and perhaps the nuget cache on your machine.

Thanks for the pointers.

I had 2.88.6 installed. Cleaned the solution and restored packages and still getting the same error. Also decompiled SkiaSharp.dll in my app and it does contain the method which the logger says is missing. Then I installed 2.88.7 and cleaned everything, still the same exact error is showing and no maps. Then I suspected something else.

Since the host application has many other plugins installed I used ProcessExplorer to see the loaded dlls in its process and there it was. Another plugin was loading an older version of SkiaSharp sooner than my addin which resulted in a dll hell situation. I removed that plugin and the maps are now showing again.
